model/incoming_webhook.go

```diff
@@ -5,39 +5,18 @@ package model
 
 import (
 	"bytes"
-	"crypto/hmac"
-	"crypto/sha1"
-	"crypto/sha256"
-	"encoding/hex"
 	"encoding/json"
-	"hash"
 	"io"
 	"net/http"
 	"regexp"
 	"strconv"
-	"strings"
 )
 
 const (
 	DEFAULT_WEBHOOK_USERNAME = "webhook"
-	WHITE_IP_LIST_SIZE       = 5 //upper limit
+	IP_WHITELIST_SIZE        = 5 //upper limit
```

Comment:
Instead of `//upper limit`, suggest `MAX_IP_WHITELIST_SIZE` for optimal clarity.

Reply:
whitelist is taken out.